musicbrainz/csv/README.md

```diff
@@ -0,0 +1,10 @@
+# MusicBrainz JSON -> CSV
+Steps:
+1.  Get the latest JSON files from https://data.metabrainz.org/pub/musicbrainz/data/json-dumps
+    *  Examples in data folder.
+2.  Run convert_to_csv.py, specify the entity type in the second argument.
```

Comment:
And the first argument is?

Reply:
And what is an entity type?

Reply:
Each database has a type like recording, label, area, genre, etc.